sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -5962,4 +5962,9 @@ ALTER TABLE person_history ADD COLUMN armedforcesrelationtype varchar(255);
 
 INSERT INTO schema_version (version_number, comment) VALUES (283, 'Add column armedforcesrelationtype #3418');
 
+ALTER TABLE contact_history ADD column multidaycontact boolean default false;
```

Comment:
No default needed for history tables because they are filled based on the values in the main tables